MiniAn/minian_main.py

```diff
@@ -366,7 +366,7 @@ def cnmf1(Y_hw_chk, intpath, A, C, C_chk, Y_fm_chk, chk, minian_parameters):
         sig = save_minian(sig_mrg.rename("sig_mrg"), intpath, overwrite=True)
 
     # Renumber unit_ids to start from 1 instead of 0
-    ids = A["unit_id"].values + 1
```

Comment:
It look like that for me doing the renumbering there don't work. But doing it in `cnmf2()` in line 412 work just before doing  `AC = compute_AtC(A, C_chk) ` .

I don't think the data I use are different but for me when I test I do the computation on binned 2x2 images